plugins/inputs/linux_cpu/linux_cpu_test.go

```diff
@@ -43,12 +43,12 @@ func TestNoCPUMetrics(t *testing.T) {
 func TestGatherCPUFreq(t *testing.T) {
 	td := t.TempDir()
 
-	require.NoError(t, os.MkdirAll(td+"/devices/system/cpu/cpu0/cpufreq", os.ModePerm))
+	require.NoError(t, os.MkdirAll(td+"/devices/system/cpu/cpu0/cpufreq", 0750))
```

Comment:
Here we probably just should add these files to the testdata/testcases instead of rewriting everything during the tests, but probably not for this PR...

Reply:
Agreed, I can look at the various tests that do this (bcache, lustre2, zfs, and linux_cpu) and follow up.